.circleci/config.yml

```
@@ -62,7 +62,7 @@ workflows:
           context:
             - browserstack-env
       - ship/node-publish:
-          publish-command: npm publish --tag beta
+          publish-command: npm publish --tag
```

Does it matter that we keep an empty `--tag`?

---

Gah yeah that'll error I bet
```suggestion
          publish-command: npm publish 
```